source/default/_patterns/00-protons/legacy/css/utc-sidebar-menu.css

```diff
@@ -1,129 +1,167 @@
-.utc-sidebar ol, .utc-sidebar ul {
-  list-style: none;
-  padding-left: 0;
+.utc-sidebar ol,
+.utc-sidebar ul {
+    list-style: none;
+    padding-left: 0;
 }
-.utc-sidebar ol li, .utc-sidebar ul li {
-  margin-bottom: 0.3em;
+
+.utc-sidebar ol li,
+.utc-sidebar ul li {
+    margin-bottom: 0.3em;
 }
-.utc-sidebar ol li ul li, .utc-sidebar ul li ul li {
-  margin: 0.3em auto;
+
+.utc-sidebar ol li ul li,
+.utc-sidebar ul li ul li {
+    margin: 0.3em auto;
 }
-.utc-sidebar ol li ul li:first-child, .utc-sidebar ul li ul li:first-child {
-  border-top: none;
+
+.utc-sidebar ol li ul li:first-child,
+.utc-sidebar ul li ul li:first-child {
+    border-top: none;
 }
-.utc-sidebar ol li ul li ul li, .utc-sidebar ul li ul li ul li {
-  margin: 0.3em auto;
+
+.utc-sidebar ol li ul li ul li,
+.utc-sidebar ul li ul li ul li {
+    margin: 0.3em auto;
 }
-.utc-sidebar ol li ul li ul li:first-child, .utc-sidebar ul li ul li ul li:first-child {
-  border-top: none;
+
+.utc-sidebar ol li ul li ul li:first-child,
+.utc-sidebar ul li ul li ul li:first-child {
+    border-top: none;
 }
+
 .utc-sidebar .sidebar-menu li {
-  position: relative;
+    position: relative;
```

Comment:
Devs should standardize tabbing/spacing formats so there are not large deltas due to tab/space changes... these tend to make it difficult to do code reviews.

Are these manually added or automatically formatted by the IDE?